modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -346,6 +346,43 @@ This example shows the `build` section of `pom.xml`:
 </build>
 ----
 
+== Publish a parent pom project and a parent pom dependant project to Exchange
+
+Parent poms are published to Exchange as custom assets. See previous section in order to publish a custom asset.
+If you want to publish a maven project, like a mule application, using the Mule Maven Plugin, and your maven project depends on a Parent Pom that has already included the exchange-mule-maven-plugin, you need to https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_.3Cinherited.3E_Tag_In_Build_Plugins[prevent the inheritance]:
```

Comment:
```suggestion
If you want to publish a Maven project, like a mule application, using the Mule Maven plugin, and your Maven project depends on a parent POMthat is already included the exchange-mule-maven-plugin, you must https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_.3Cinherited.3E_Tag_In_Build_Plugins[prevent the inheritance]:
```

Reply:
This is not why i actually mean. 
The desired output will be something like: "if your maven project uses the mule-maven-plugin, and depends on a parent on that already includes the exchange-mule-maven-plugin, you must break the inheritance"